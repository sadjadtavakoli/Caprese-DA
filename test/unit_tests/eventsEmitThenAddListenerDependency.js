var events = require('events');
var eventEmitter = new events.EventEmitter();

function listner1() {
    //   do nothing 
}

function adderFunction() {
    eventEmitter.addListener('connection2', listner1);
}

function emitter(){
    eventEmitter.emit("connection2", "args")
}

emitter()
adderFunction()
