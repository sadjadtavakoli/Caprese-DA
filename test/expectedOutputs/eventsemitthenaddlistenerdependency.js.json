{"emitter-test/unit_tests/eventsemitthenaddlistenerdependency.js-12-14":{"impacted":["test/unit_tests/eventsemitthenaddlistenerdependency.js"]},"adderFunction-test/unit_tests/eventsemitthenaddlistenerdependency.js-8-10":{"impacted":["test/unit_tests/eventsemitthenaddlistenerdependency.js"]}}